tests/trestle/core/models/update_action_test.py

```diff
@@ -36,7 +36,7 @@ def test_update_action(sample_target_def):
         }
     )
 
-    sub_element_path = ElementPath('target-definition.metadata')
+    sub_element_path = ElementPath('t-definition.metadata')
```

Comment:
This doesn't look right...

Reply:
You are right. I've updated, however, the only part queried is the leaf of the element path in this function so this method is okay.